tests/cli/test_repos_sanity.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+
+rlJournalStart
+    rlPhaseStartSetup
+        repodir_backup=$(mktemp -d composerrepos-XXXXX)
+
+        if [[ -z "$CLI" || "$CLI" == "./src/bin/composer-cli" ]]; then
+            composer_params=$(ps ax | grep -v grep  | grep -m 1 -Po 'bin/lorax-composer \K.*')
+        else
+            composer_params="/var/lib/lorax/composer/blueprints"
+        fi
+        composer_stop
+
+        rlRun -t -c "mv /var/lib/lorax/composer/repos.d/* $repodir_backup"
+        rlRun -t -c "mv /etc/yum.repos.d /etc/yum.repos.d.disabled"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Run lorax-composer with --no-system-repos option"
```

Comment:
If `/var/lib/lorax/composer/repos.d/` is removed then the next few lines will report PASS.